### DISCUSSION TITLE
Definition of Done: Product

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#52] [How we work]
I propose the following definition of done for product work on the project (signaling the design is ready to be handed over to design (and, in rare cases, directly to development)):

- [ ] hypothesis about why the feature is needed (ideally supported by qualitative and quantitative data)
- [ ] tight scope of the feature (MVP!) written as user stories or similar language
- [ ] what data should be collected (and how) to validate whether the feature MVP effectively addresses the hypothesis
- [ ] how, when, and who should discuss the collected data to learn from the feature MVP
- [ ] the team considers the pitch [safe to try and consents](https://www.perplexity.ai/page/safe-to-try-experiments-in-org-WwEa_qOrRzGaHSl0N5Ai1g) to working on it

<details><summary>Comments</summary>
<p>

- This also means that any level of polish or iteration, such as adding haptic feedback and sound effects, should be written about, discussed, and then sent on to design/development. That should protect both design and development from me erratically getting obsessed with details and polish.

</p>
</details>

**Comment by aripuguh** [comment]
"signaling the design is ready to be handed over to design...production or process maybe?" :)
